diff -urN aa/libgtk-java-2.4.5/src/jni/org_gnu_gtk_ActionGroup.c libgtk-java-2.4.5/src/jni/org_gnu_gtk_ActionGroup.c
--- aa/libgtk-java-2.4.5/src/jni/org_gnu_gtk_ActionGroup.c	2004-10-21 22:48:12.000000000 +0200
+++ libgtk-java-2.4.5/src/jni/org_gnu_gtk_ActionGroup.c	2004-11-02 01:09:24.994074264 +0100
@@ -25,6 +25,19 @@
     jmethodID methodID;  /* methodID of callback method */
 } CallbackInfo;
 
+static void internalHandleActionCallback(gpointer action, CallbackInfo *cbi)
+{
+	jvalue *jargs = g_new(jvalue, 1);
+
+	if (cbi == NULL) {
+	g_critical("Java-GNOME - unable to determine the callback method\n");
+	return;
+	}
+
+	jargs->i = (jint)action;
+	(* cbi->env)->CallVoidMethodA(cbi->env, cbi->obj, cbi->methodID, jargs);
+}
+
 /*
  * Callback method to handle all signals.
  */
@@ -39,19 +52,6 @@
 	internalHandleActionCallback((gpointer) action, cbi);
 }
                                                                                                                                                
-static void internalHandleActionCallback(gpointer action, CallbackInfo *cbi)
-{
-	jvalue *jargs = g_new(jvalue, 1);
-
-	if (cbi == NULL) {
-	g_critical("Java-GNOME - unable to determine the callback method\n");
-	return;
-	}
-
-	jargs->i = (jint)action;
-	(* cbi->env)->CallVoidMethodA(cbi->env, cbi->obj, cbi->methodID, jargs);
-}
-
 /*
  * Class:     org_gnu_gtk_ActionGroup
  * Method:    gtk_action_group_get_type
